ci-operator/step-registry/osd-ccs/fleet-manager/osd-ccs-fleet-manager-workflow.yaml

```diff
@@ -13,8 +13,9 @@ workflow:
     - ref: osd-ccs-fleet-manager-sc-provision
     - chain: rosa-aws-sts-provision-byo-vpc
     - ref: osd-ccs-conf-idp-htpasswd-multi-users
+    - ref: osd-ccs-fleet-manager-test
```

Comment:
Suggest to put "osd-ccs-fleet-manager-test" right after "osd-ccs-fleet-manager-sc-provision".
Because if any FM configuration test case failed and exit 1, the provisioning steps of hosted cluster is not necessary and can be avoided.

Reply:
done